src/compiler/utilities.ts

```diff
@@ -1551,6 +1559,10 @@ namespace ts {
     }
 
    export function getJSDocs(node: Node): (JSDoc | JSDocTag)[] {
+        if (isJSDocTypedefTag(node) && isJSDoc(node.parent)) {

```

Comment:
Why is this needed? it is not clear what code path would get us here from the change.